src/Views/Filter/Location.cshtml

```diff
@@ -46,9 +46,15 @@
                         <div class="form-group">
                             <label class="form-label" for="radius">Within</label>
                             <select class="form-control" id="radius" name="rad">
-                                @foreach(var option in Enum.GetValues(typeof(RadiusOption)))
+                                @foreach(int option in Enum.GetValues(typeof(RadiusOption)))
                                 {
-                                    <option value="@((int) option)">@((int) option) miles</option>
+                                    var defaultSelectedValue = 20;
+                                    if(option == defaultSelectedValue){
```

Comment:
This will override any previous selection with 20 miles, while other inputs will reflect previous selections. For consistency, you'll have to check whether the RadiusOption property is null, i.e. if (Model.RadiusOption == null ? option == defaultSelectedValue : option == (int) Model.RadiusOption) {

Reply:
👍 @defong are you able to help?

Reply:
Will have a look.